assets/css/main.css

```
@@ -0,0 +1,877 @@
+}
+
+.culture-navigation {
+    align-items: center;
+    color: #949ba4;
+    display: flex;
+    justify-content: space-between;
+}
+
+.culture-navigation label {
+    background-color: rgba(148, 155, 164, .1);
+    border: none;
+    border-radius: 50%;
+    cursor: pointer;
+    display: none;
+    font-size: 24px;
+    height: 40px;
+    margin-right: 40px;
+    padding: 0;
+    pointer-events: auto;
+    transition: all .3s;
+    width: 40px;
+}
+
+#culture-item1:checked ~ .culture-navigation label[for="culture-item4"].culture-slide-prev,
+#culture-item1:checked ~ .culture-navigation label[for="culture-item2"].culture-slide-next,
+#culture-item2:checked ~ .culture-navigation label[for="culture-item1"].culture-slide-prev,
+#culture-item2:checked ~ .culture-navigation label[for="culture-item3"].culture-slide-next,
+#culture-item3:checked ~ .culture-navigation label[for="culture-item2"].culture-slide-prev,
+#culture-item3:checked ~ .culture-navigation label[for="culture-item4"].culture-slide-next,
+#culture-item4:checked ~ .culture-navigation label[for="culture-item3"].culture-slide-prev,
+#culture-item4:checked ~ .culture-navigation label[for="culture-item1"].culture-slide-next
+{
+    align-items: center;
+    display: flex;
+    justify-content: center;
+}
+
+.culture-item-title {
+    cursor: pointer;
+    font-size: 14px;
+    font-weight: 700;
+    height: 75px;
+    margin-bottom: 20px;
+    margin-top: 30px;
+}
+
+.culture-item-title:hover {
+    color: #c01111;
+}
+
+.culture-content {
+    position: relative;
+    z-index: 0;
+}
+
+.culture-content::before {
+    background-color: rgba(228, 228, 228, .4);
+    border-radius: 50%;
+    content: '';
+    height: 20px;
+    left: 2px;
+    position: absolute;
+    top: 12px;
+    transform: translate(-50%, -50%);
+    width: 20px;
+    z-index: -1;
+}
+
+.culture-item-description {
+    color: #949ba4;
+    font-family: 400;
+    font-size: 14px;
+    height: 75px;
+    line-height: 1.8;
+    margin-bottom: 20px;
+    overflow: hidden;
+}
+
+.culture-item-detail {
+    color: #c01111;
+    font-weight: 700;
+    line-height: 1;
+    display: flex;
+    align-items: center;
+    font-size: 14px;
+}
+
+.culture-item-detail .button-icon {
+    background-color: rgba(192, 17, 17, .1);
+    border-radius: 50%;
+    height: 24px;
+    margin-left: 12px;
+    width: 24px;
+}
+
+.culture-btn {
+    margin: 75px auto 0;
+    width: 366px;
+}
+
+.culture-slides  {
+    animation: counterSlider 20s infinite linear;
+    display: flex;
+}
+
+@keyframes counterSlider {
+    0% { transform: translateX(0%); } 
+    30% { transform: translateX(0%); }
+  
+    35% { transform: translateX(-33.33%); }
+    65% { transform: translateX(-33.33%); } 
+  
+    70% { transform: translateX(-66.66%); }
+    100% { transform: translateX(-66.66%); }
+}
+
+.culture-navigation label:hover {
+    background-color: #c01111;
+    color: #fff;
+}
+
+.culture-image {
+    position: relative;
+}
+
+.culture-event--label {
```

```suggestion
.culture-event__label {
```
Về cách đặt tên class, em đang chưa đặt theo quy tắc BEM, e fix tương tự cho 
 1 số selector phía trên. Comment này em fix trong pull sau cũng được nha